src/Sql/Sql/help/New-AzSqlInstanceLink.md

```diff
@@ -0,0 +1,300 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version:
+schema: 2.0.0
+---
+
+# New-AzSqlInstanceLink
+
+## SYNOPSIS
+Creates a new Azure SQL Managed Instance Link.
+
+## SYNTAX
+
+### CreateByNameParameterSet (Default)
+```
+New-AzSqlInstanceLink [-ResourceGroupName] <String> [-InstanceName] <String> [-Name] <String>
+ -PrimaryAvailabilityGroupName <String> -SecondaryAvailabilityGroupName <String> -TargetDatabase <String>
+ -SourceEndpoint <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### CreateByParentObjectParameterSet
+```
+New-AzSqlInstanceLink [-Name] <String> -PrimaryAvailabilityGroupName <String>
+ -SecondaryAvailabilityGroupName <String> -TargetDatabase <String> -SourceEndpoint <String>
+ [-InstanceObject] <AzureSqlManagedInstanceModel> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzSqlInstanceLink** cmdlet creates an Azure SQL Managed Instance Link by join Distributed Availability Group on SQL Server based on the parameters passed.
+
+## EXAMPLES
+
+### Example 1: Create a new Managed Instance Link
+```powershell
+PS C:\> New-AzSqlInstanceLink -ResourceGroupName "ResourceGroup01" -InstanceName "Instance01" -Name "Link01" -PrimaryAvailabilityGroupName "Link01PrimaryAG" -SecondaryAvailabilityGroupName "Link01SecondaryAG" -TargetDatabase "Link01DB" -SourceEndpoint "TCP://SERVER01:7022"		
+ResourceGroupName              : ResourceGroup01
+InstanceName                   : Instance01
```

Comment:
@ostojicb  - please modify these examples here to use these values:
- ManagedInstance 01
- Database01
- port 5022